app/views/game/init.haml

```diff
@@ -1,21 +1,64 @@
-= form_tag(:turn, id: 'turn', remote: true) do
-  = hidden_field_tag 'id'
-  = submit_tag 'ok', class: 'turn_submit'
+.container
+  .block-form
+    = form_tag(:turn, id: 'turn', remote: true, class: 'form') do
+      = hidden_field_tag 'id'
+      = submit_tag 'ok', class: 'turn_submit'
+      %h1.page-header
+        .bg-primary
+          PLAYING
+          %span#player= @players.current.name
+        .bg-primary
+          WITH SYMBOL
+          %span#symbol= @players.current.symbol
 
-  #current_player
-    %div
-      = @players.current.name
-  #board
-    %table
-      %tr#row1
-        %td.square#0
-        %td.square.v#1
-        %td.square#2
-      %tr#row2
-        %td.square.h#3
-        %td.square.v.h#4
-        %td.square.h#5
-      %tr#row3
-        %td.square#6
-        %td.square.v#7
-        %td.square#8
+      .block-form
+        .modal.fade{:role => "dialog", :tabindex => "-1"}
+          .modal-dialog
+            .modal-content
+              .modal-header
+                %button.close{"aria-label" => "Close", "data-dismiss" => "modal", :type => "button"}
+                  %span{"aria-hidden" => "true"} ×
+                %h4.modal-title Game over!
+              .modal-body
+                %p#over-message
+              .modal-footer
+                %button.btn.btn-default{"data-dismiss" => "modal", :type => "button"} Close
+                %button.btn.btn-primary.new-game{:type => "button"} New game!
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace